packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -185,6 +277,34 @@ export class ProductCard extends PureComponent {
             break;
         }
 
+        if (type_id === BUNDLE) {
+            if (price_view === 'PRICE_RANGE' && dynamic_price) {
+                return (
+                    <div block="ProductCard" elem="PriceWrapper">
+                        { this.renderBundlePriceBadge('bottom') }
+                        <ProductPrice
+                          price={ price_range }
+                          mix={ { block: 'ProductCard', elem: 'Price' } }
+                        />
+                        { this.renderBundlePriceBadge('top') }
+                        { this.renderMaxBundlePrice() }
+                    </div>
+                );
+            }
+
+            if (price_view === 'AS_LOW_AS') {
```

Comment:
Please move the constants to the config.
Or reuse them. Do the search by the code first.